internal/verifier/migration_verifier_test.go

```diff
@@ -1476,10 +1489,12 @@ func (suite *MultiDataVersionTestSuite) TestVerifierWithFilter() {
 	// Wait for one generation to finish.
 	<-checkDoneChan
 	status := waitForTasks()
-	suite.Require().Equal(VerificationStatus{TotalTasks: 2, FailedTasks: 0, CompletedTasks: 2}, *status)
+	suite.Require().Greater(status.CompletedTasks, 1)
+	suite.Require().Greater(status.TotalTasks, 1)
```

Comment:
Why is it Greater now instead of Equal 2? I'm not very familiar with the verifier

Reply:
I think it was 2 because there's 1 task to verify a partition and another task to verify the collection. Now that we're changing this test to create many partitions, I think it's hard to test the exact number of tasks / partitions in this high-level test.